components/EntireScheme.svelte

```diff
@@ -4,6 +4,10 @@
 
   export let authorityName;
 
+  // Strings
+  let otherFiles = [];
```

Comment:
If we actually decide to merge this, I'd want to clean up the code in a few ways. Maybe store processed JSON instead of strings, stop special-casing the "load 1" vs "load many" cases